RoddyCore/test/src/de/dkfz/roddy/config/FilenamePatternTest.groovy

```diff
@@ -110,7 +112,7 @@ class FilenamePatternTest {
     @Test
     @Ignore("Fix. Use true temp dirs. NullPointerException.")
     public void testJobCreationWithFileUsingToolIDForNamePattern() {
-        MockupExecutionContextBuilder.createMockupJobManager()
+        contextResource.createMockupJobManager()
```

Comment:
I wonder, if contextResource is the right name. It also is used for other cases than the context creation.